effekt/jvm/src/test/scala/effekt/StdlibTests.scala

```diff
@@ -23,7 +23,9 @@ abstract class StdlibChezTests extends StdlibTests {
   override def ignored: List[File] = List(
     // Not implemented yet
     examplesDir / "stdlib" / "bytearray",
-    examplesDir / "stdlib" / "io"
+    examplesDir / "stdlib" / "io",
+    examplesDir / "stdlib" / "stream" / "characters.effekt",
+    examplesDir / "stdlib" / "stream" / "fuse_newlines.effekt"
```

Comment:
We should make a follow up issue for this (it's just Valgrind problems with strings...)

Reply:
=> #715.